server_status/__init__.py

```diff
@@ -1,3 +1,3 @@
 # -*- coding: utf-8 -*-
 # pylint: disable=missing-docstring
-__version__ = '0.3.2'  # pragma: no cover
```

Comment:
Can we wait to bump the version until after the release-candidate branch is already cut to align with our release process? https://github.com/mitodl/release-script/blob/master/README.md